crates/wash-cli/src/cmd/dev/session.rs

```diff
@@ -431,4 +403,64 @@ impl WashDevSession {
 
         Ok((nats_child, wadm_child, wasmcloud_child))
     }
+
+    async fn get_host_id(log_path: PathBuf) -> anyhow::Result<String> {
```

Comment:
I'm not sure if this is the best place for this, organizationally. Is it better to put it as a function right in the module? Is this always accessible through `WashDevSession`? I dunno rust things.

Reply:
If it's not using anything from the struct itself, take it out of the impl.